sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/samples/host.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "2.0",
+    "logging": {
+        "logLevel": {
+            "Azure": "Information"
+        },
+        "applicationInsights": {
+            "samplingExcludedTypes": "Request",
+            "samplingSettings": {
+                "isEnabled": true
+            }
+        }
+    },
+    "StorageConnection": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;"
```

Comment:
Is this a well-known looked down connection?

Reply:
Yep, Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq

Reply:
Cool yes it is for Azurite https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite.